src/components/Home.js

```diff
@@ -45,8 +46,8 @@ const Home = () => {
   return (
     <div className="App">
       <header
-        className="App-header"
-        style={{ minHeight: issuesList || menu === 2 ? '60vh' : '100vh' }}>
+        className="App-header sticky top-0 left-0 right-0 z-[2]"
+        style={{ minHeight: issuesList || menu === 2 ? '30vh' : '100vh' }}>

```

Comment:
After adding '30vh' in place of '60vh' the UI of our app is not looking good if nothing loads in the issue component.
![Screenshot 2022-10-02 at 21-00-16 Issue Finder](https://user-images.githubusercontent.com/17983225/193462294-e51e2ddc-17f6-4d67-88a8-58591c865d04.png)

You will clearly see a lot of blank space below our issue app.